core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-editor-modal.component.ts

```diff
@@ -160,7 +160,8 @@ export class QuestionSuggestionEditorModalComponent
   setDifficultyString(skillDifficulty: number): void {
     // This throws "Object is possibly undefined." The type undefined
     // comes here from Object dependency. We need to suppress this
-    // error because of strict type checking.
+    // error because of strict type checking. We are sure that
```

Comment:
I don't quite understand what object the error message refers to here. skillDifficulty is already a number above (in the method signature) so it can't be referring to that, right?